pkg/kapp/clusterapply/change_set_view.go

```diff
@@ -38,7 +38,7 @@ func (v *ChangeSetView) Print(ui ui.UI) {
 		}
 	}
 
-	v.changesView = &ChangesView{ChangeViews: v.changeViews, Sort: true}
+	v.changesView = &ChangesView{ChangeViews: v.changeViews, Sort: true, countsView: NewChangesCountsView()}
```

Comment:
On line number 49, should we call the `v.changesView.Summary()` function only when `v.opts.Summary` is true? 
Based on the comment, it is assumed that `Print()` function is used before `Summary()`, but when `--diff-summary` is set to false, `Print()` is not called.